### PR TITLE
Template changes

### PR DESCRIPTION
**Pull request by thomasridd**
Merged template changes from https://github.com/ministryofjustice/hmpps-template-typescript

### CODE REVIEW

File: helm_deploy/hmpps-template-typescript/.helmignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```
Comment (brightonsbox) [inline helm_deploy/hmpps-template-typescript/.helmignore:1]: delete the `/hmpps-template-typescript` directory

File: helm_deploy/hmpps-template-typescript/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
```
Comment (brightonsbox) [inline helm_deploy/hmpps-template-typescript/Chart.yaml:1]: delete the /hmpps-template-typescript directory

File: helm_deploy/hmpps-template-typescript/.helmignore (hunk 1 above)
Comment (thomasridd) [reply helm_deploy/hmpps-template-typescript/.helmignore]: done

File: helm_deploy/hmpps-template-typescript/Chart.yaml (hunk 2 above)
Comment (thomasridd) [reply helm_deploy/hmpps-template-typescript/Chart.yaml]: done

**Comment by thomasridd** [comment]
responded to comments and done a more thorough review of ensuring correct existing updates don't get wiped

File: .github/workflows/rename_template_project_pr.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: rename-project-create-pr
```
Comment (brightonsbox) [inline .github/workflows/rename_template_project_pr.yml:1]: This can be deleted, it's no longer a template app

File: rename-project.bash [hunk 4]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```
Comment (brightonsbox) [inline rename-project.bash:1]: Can also be deleted

File: server/data/hmppsAuthClient.ts [hunk 5]
```diff
@@ -48,14 +55,14 @@ export default class HmppsAuthClient {
   }
 
   getUser(token: string): Promise<User> {
-    logger.info(`Getting user details: calling HMPPS Auth`)
-    return HmppsAuthClient.restClient(token).get({ path: '/api/user/me' }) as Promise<User>
+    logger.info('Getting user details: calling HMPPS Auth')
+    return HmppsAuthClient.restClient(token).get<User>({ path: '/api/user/me' })
```
Comment (brightonsbox) [inline server/data/hmppsAuthClient.ts:59]: Maybe a separate ticket, but we should be pointing to Manage Users API for /user/me(/roles) etc.

Comment (brightonsbox) [reply server/data/hmppsAuthClient.ts:59]: (See the work that Ravi's done recently)

Comment (thomasridd) [reply server/data/hmppsAuthClient.ts:59]: Yep - perhaps we should be making a PR to the template app for that one

File: rename-project.bash (hunk 4 above)
Comment (thomasridd) [reply rename-project.bash]: removed

File: .github/workflows/rename_template_project_pr.yml (hunk 3 above)
Comment (thomasridd) [reply .github/workflows/rename_template_project_pr.yml]: removed

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**